storage-model/src/test/java/no/unit/nva/publication/storage/model/UserInstanceTest.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.publication.storage.model;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.testing.PublicationGenerator;
+import org.junit.jupiter.api.Test;
+
+class UserInstanceTest {
+
+    @Test
+    void shouldReturnUserInstanceFromPublication() {
```

Comment:
The methods are used in another module. These tests are for test coverage of the module